pom.xml

```diff
@@ -9,9 +9,22 @@
 	<version>1.0.0-BUILD-SNAPSHOT</version>
 	<properties>
 		<java-version>1.6</java-version>
-		<org.springframework-version>3.1.1.RELEASE</org.springframework-version>
+		<org.springframework-version>4.2.4.RELEASE</org.springframework-version>
+		<org.springframework-data-version>1.8.4.RELEASE</org.springframework-data-version>
+		<org.springframework-boot-parent>1.3.2.RELEASE</org.springframework-boot-parent>
 		<org.aspectj-version>1.6.10</org.aspectj-version>
 		<org.slf4j-version>1.6.6</org.slf4j-version>
+        <log4j-version>1.2.15</log4j-version>
+        <javax.servlet-api-version>2.5</javax.servlet-api-version>
+        <javax.servlet.jsp-version>2.1</javax.servlet.jsp-version>
+        <javax.servlet-jstl-version>1.2</javax.servlet-jstl-version>
+        <com.twitter-version>2.2.0</com.twitter-version>
+        <junit-version>4.7</junit-version>
+        <!-- Program Args-->
+        <consumer.key>YcXvnr93d2JI88vCHKCPH3rF8</consumer.key>
```

Comment:
So, I spent approx 1 hour on this with no easy results.. Maven has a properties plugin that it uses to auto insert properties from a file during initialization. However pathing of the file is confusing and I couldnt get it to work. Can you try this ? Until then ill let this be like this,